.pipelines/templates/template-deploy-shared-env.yml

```diff
@@ -1,12 +1,16 @@
 parameters:
-  workingDirectory: ''
+  subscriptionID: ''
+  azureDevOpsJSONSPN: ''
 steps:
 - script: |
     set -e
-    cd ${{ parameters.workingDirectory }}
 
     . secrets/env
+    export DATABASE_NAME=v4-e2e-V$BUILD_BUILDID
```

Comment:
Yes, it does not. But this is due to mess in Shell scripts. This will be refactored once we got rid of shell magic :/ 
TL;DR: `deploy-shared-env.sh` and `run-rp-and-e2e.sh` should not share as much copy-paste as they do to avoid something like this...